alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/pom.xml

```diff
@@ -27,6 +27,9 @@
     <swagger-codegen-maven-plugin.groupId>io.swagger.codegen.v3</swagger-codegen-maven-plugin.groupId>
     <swagger-codegen-maven-plugin.version>3.0.36</swagger-codegen-maven-plugin.version>
     <exec-maven-plugin.version>3.0.0</exec-maven-plugin.version>
+    <alfresco-deployment-service.version>7.12.0-alpha.22</alfresco-deployment-service.version>
+    <alfresco-modeling-service.version>7.12.0-alpha.11</alfresco-modeling-service.version>
+    <alfresco-process.version>7.12.0-alpha.6</alfresco-process.version>
```

Comment:
Shouldn't this be updated automatically?

Reply:
Yes, but properties must be present in the pom to be updated, at the moment I have put it a default value

Reply:
@pmartinezga as you can see here the version should be updated https://github.com/Alfresco/alfresco-process-sdk/pull/57/files